### PR TITLE
Add some missing jsdoc comments and add some comment related eslint rules

### PR DESCRIPTION
**Pull request by xlexi**

### CODE REVIEW

File: test/v2/support/rescue.js [hunk 1]
```diff
@@ -3,6 +3,12 @@ const { POST, Request } = require('../../../api/classes/Request')
 
 const HTTP_CREATED = 201
 
+/**
+ * Create a rescue payload
+ * @param auth authentication credentials
+ * @param r request
```
Comment (GeekyDeaks) [inline test/v2/support/rescue.js]: `r` is actually the rescue details in the format expected by the `/rescue` endpoint

File: bin/createTestDB.js [hunk 2]
```diff
@@ -49,8 +49,12 @@ const seed = {
   alpha: 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
 }
 
-// predictable simple semi psuedo random generator
-// https://stackoverflow.com/a/19303725/3785845
+/**
+ * predictable simple semi psuedo random generator (https://stackoverflow.com/a/19303725/3785845)
+ * @param seed the random seed
+ * @param m prand seed
```
Comment (GeekyDeaks) [inline bin/createTestDB.js]: I'm an idiot - I need to pick a better const name for the default value of `m`.  i.e. DEFAULT_PRAND_BIAS - should probably then change the variable to `bias` also...

File: test/v2/support/rescue.js (hunk 1 above)
Comment (trezy) [reply test/v2/support/rescue.js]: Can we get out of the habit of using single character variable names? It's not going to make the code any faster, but it *will* make the code less readable, even with JSDoc.

Comment (GeekyDeaks) [reply test/v2/support/rescue.js]: yeah - sorry :D

Comment (trezy) [reply test/v2/support/rescue.js]: YOU BETTER BE SORRY.

Comment (trezy) [reply test/v2/support/rescue.js]: 🤗

**Pull request merged**